src/main/java/com/charu/Tasks.java

```diff
@@ -0,0 +1,51 @@
+package com.charu;
+
+class Tasks{
+
+
+
+    private int id;
+
+    private String  name;
+
+    private String status;
```

Comment:
status should be ENUM - `TODO`, `IN PROGRESS`, `DONE`

Reply:
addressed